packages/frontend/src/pages/Home.jsx

```diff
@@ -18,12 +18,21 @@ export default observer(() => {
   }
   return (
     <>
-      <video autoPlay muted loop poster playsInLine>
-        <source
-          src={require('../../public/unirep-ceremony-hero.mp4')}
-          type="video/mp4"
-        />
-      </video>
+      {ui.isMobile ? (
```

Comment:
this is good

Reply:
hmm idk i didn't see that in the mobile simulator, yeah it's kinda wonky on the phone. i'll try to fix it first.